packages/contracts/contracts/universal/ITokenMultiSigWallet.sol

```diff
@@ -0,0 +1,101 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+/**
+ * @title ITokenMultiSigWallet.sol.sol
+ * @notice Interface for contracts of a token based multi-signature wallet.
+ */
+interface ITokenMultiSigWallet {
+    /**
+     * @notice Emitted when anyone submit a transaction.
+     *
+     * @param submitter     Owner of address that submitted a transaction.
+     * @param transactionId Index of transaction submitted.
```

Comment:
Currently, it is not the index, unique id. Please change this comment and below comments also to transaction id.